spec/unit/block_score/actions/delete_spec.rb

```diff
@@ -3,9 +3,35 @@
 RSpec.describe BlockScore::Actions::Delete do
   describe '#delete' do
     subject(:candidate) { create(:candidate) }
-    before { candidate.delete }
+    before do
+      expect(candidate.delete).to be true
+    end
 
-    it { is_expected.not_to be_persisted }
-    it { expect(candidate.deleted).to be true }
+    its(:persisted?) { is_expected.to be false }
+    its(:deleted?) { is_expected.to be true }
+
+    context 'previously deleted' do
+      it 'was already deleted' do
+        expect(candidate.delete).to be false
+      end
+    end
+  end
+
+  describe '#delete!' do
+    subject(:candidate) { create(:candidate) }
+    before do
```

Comment:
When you get to that point lmk. I have a few style guidelines wrt spec layout that can make things more consistent, and improve the readability.